lambda/src/main/scala/pricemigrationengine/handlers/EstimationHandler.scala

```diff
@@ -38,9 +41,38 @@ object EstimationHandler extends App with RequestHandler[Unit, Unit] {
         )
     } yield ()
 
+  /*
+   * Earliest start date spread out over 3 months.
+   */
+  private[handlers] def spreadEarliestStartDate(
+      subscription: ZuoraSubscription,
+      invoicePreview: ZuoraInvoiceList
+  ): ZIO[AmendmentConfiguration with Random, ConfigurationFailure, LocalDate] = {
+
+    def earliestStartDateForAMonthlySub(config: AmendmentConfig) =
+      for {
+        randomFactor <- random.nextIntBetween(0, 3)
```

Comment:
I think this will spread the subs over the next 4 months?

Reply:
The max value is exclusive: https://github.com/zio/zio/blob/master/core/shared/src/main/scala/zio/random/package.scala#L111

Reply:
Ah sorry :)